pages/docs/reference/data-science-overview.md

```diff
@@ -7,137 +7,44 @@ title: "Kotlin for Data Science"
 
 # Kotlin for Data Science
 
-![Kotlin for data science]({{ url_for('asset', path='images/landing/data-science/data-science-overview.png')}})
+From building ETL pipelines to productionizing machine learning models, Kotlin can be a great choice for 
+working with data:
+* Static typing and null safety help create reliable, maintainable code that is easy to troubleshoot. 
+* Kotlin is concise, readable and easy to learn. 
```

Comment:
I'd put this in the first place: it's the simplest idea in the list and it's about starting the user journey (`easy to learn`)

Reply:
Fixed in 0589f3228c213d9072011459e408d442132c4108